github_connector_odoo/models/odoo_author.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class OdooAuthor(models.Model):
```

Comment:
https://github.com/OCA/interface-github/pull/1/files#r93159474

Reply:
I think the problem is that the models are called odoo.*

This *can* be a problem when you have naming conflicts.
Just tested this by making a model called odoo.fields, there are no exceptions or errors, and Odoo behavior is not changed. Technically there is no issue in calling the models like this.

Reply:
Then let's move forward